commands.json

```diff
@@ -2854,6 +2855,21 @@
             "stale"
         ]
     },
+    "COMMAND GETKEYSANDFLAGS": {
+        "summary": "Extract keys given a full Redis command",
```

Comment:
```suggestion
        "summary": "Extract keys and flags given a full Redis command",
```

Reply:
@itamarhaber should i remind you this is an auto-generated file?
i can apply your suggestion on the source, but i'm too lazy, and i think it's clear without it..